ui/app/routes/vault/cluster/dashboard.js

```diff
@@ -6,10 +6,11 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 import { hash } from 'rsvp';
+// eslint-disable-next-line ember/no-mixins
+import ClusterRoute from 'vault/mixins/cluster-route';
 
-export default class VaultClusterDashboardRoute extends Route {
-  @service store;
-  @service version;
+export default Route.extend(ClusterRoute, {
```

Comment:
I'd be curious if @zofskeez had any thoughts here. It'd be nice to make a decorator instead of adding tech debt, buuut I know that's an undertaking and likely outside of the scope here. 

I think it makes sense to use the mixin and update later.

Reply:
I think option 2 is reasonable considering the add to the scope but I would suggest making this route a native class and extending the mixin using `EmberObject.extend` so that it's one less thing to update when we do get around to refactoring the `ClusterRoute` mixin.

```js
import EmberObject from '@ember/object';

export default class DashboardRoute extends EmberObject.extend(ClusterRoute) {}
```

Here's a [link to the docs](https://guides.emberjs.com/release/upgrading/current-edition/native-classes/#toc_mixins) for more information.